zomes/blog/capabilities/main/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "capability": {
+    "membrane": "agent"
+  },
+
+  "functions": [
```

Comment:
just looking at https://github.com/faaslang/faaslang#faaslang-resource-definition for inspiration

they are trying to do almost the same thing as us... why don't we follow the standard, or something close to?

- support `description` for the function, inputs and outputs
- change `inputs` to `params`
- change `outputs` to `returns`
- support `defaultValue`
- support `context`
- support all the `type`s they support

differences:

- we can't support `bg` because we don't have "running in the background"
- we can't support `charge` because that's at the holo level, not holochain
- we don't need `format`
  - languages are at the zome level for us (or are they?)
  - we don't offer `async` (or do we?)
- we have an extra `type` here called `hash`

Reply:
Okay for me.
I do wonder how much we would want to have our own "community standard" more in line with P2P and Ceptr thinking.

Reply:
As we discussed in today's meeting and how I've put it in the PR description, I think this should be a next PR so that we can merge this one fairly soon...